about/design-goals.md

```diff
@@ -0,0 +1,57 @@
+# CloudEvents Specification Design Goals
+
+CloudEvents are typically used in a distributed system to allow for services to
+be loosely coupled during development, then may be configured with
+tight-coupling for specific applications or dynamically configured at runtime.
+
+The goal of the CloudEvents specification is to define interoperability of event
+systems that allow cloud providers, third-party tools and services to provide:
+- new event sources
+- new event consumers
+- observability, such as monitoring, debugging and visualization,
+- developer tools and libraries
+
```

Comment:
I think it is also an important design goal to define the event attributes in a way they can be added as extension to existing protocol headers like it is done for HTTP. So it's not about introducing a full-fledged event layer.

Reply:
+1